### ISSUE TITLE
Errors while generating a fresh project

### ISSUE THREAD

**Issue by ConKuk** [bluelibs/bluelibs#318]
**Describe the bug**
When trying to generate a new project with 'x' and then running `npm run blueprint:generate`, it results in the following error:

`npm ERR! code ERESOLVE`
`npm ERR! ERESOLVE unable to resolve dependency tree`
`npm ERR! `
`npm ERR! While resolving: api@1.0.0`
`npm ERR! Found: passport@0.6.0`
`npm ERR! node_modules/passport`
`npm ERR!   passport@"^0.6.0" from the root project`
`npm ERR! `
`npm ERR! Could not resolve dependency:`
`npm ERR! peer passport@"^0.4.1" from @bluelibs/apollo-security-bundle@1.0.3`
`npm ERR! node_modules/@bluelibs/apollo-security-bundle`
`npm ERR!   @bluelibs/apollo-security-bundle@"^1.0.0" from the root project`
`npm ERR! `
`npm ERR! Fix the upstream dependency conflict, or retry`
`npm ERR! this command with --force, or --legacy-peer-deps`
`npm ERR! to accept an incorrect (and potentially broken) dependency resolution.`

I have checked github, this repo has the dependency set on 0.6.0.
But the files I get from npm for bluelibs/apollo-security-bundle still have the dependency set on 0.4.1.

**To Reproduce**
1. Run x
2. Generate new project <projectName>
3. cd into <projectName>
4. Run `npm run blueprint:generate`

**Expected behavior**
The project should generate without errors

**Actual behavior**
The error posted above.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.4

**Additional context**
I tried running it with --force.
The project generates, but when I try to run `npm run start:api` and then `npm run start:admin`, I have a new issue.

Run API:
![image](https://user-images.githubusercontent.com/4126743/184108334-d6543396-f0b0-4a1d-a993-284e76bf19b6.png)
Not sure if it's supposed to get stuck on `timing config:load:flatten Completed in #ms`, but it does. ( The #ms part always changes )

Run admin:
![image](https://user-images.githubusercontent.com/4126743/184108531-55330bbd-1210-4db9-b87f-12be352098f8.png)

Hope this helps figuring it out.

**Comment by y-moufakkr** [comment]
Hello, the issue that u raised about passport version in apollo-security should be fixed, 
however to unblock u for now  make sure to delete your microservices or at least there `node_modules` and `package_lock.json`, and also on your root folder, and do a` npm cache clean --f` => `npm run i --f` =>` npm run blueprint:generate --f` , 
also this seems to be a npm/node versions issue, make sure to have a node version > v16, 
example of working version `node :v16.14.0`, `npm:8.3.0`

**Comment by ConKuk** [comment]
Thank you for the quick response.

I have already installed using --f. however as you mentioned it was a node version issue.
I upgraded my node to v16 ( was using v14 ) and now the project loads fine.

**Comment by theodorDiaconu** [comment]
Are we broken on Node 14?

**Comment by y-moufakkr** [comment]
this is the first time I see such an issue on v14, but x-ui seems to throw a lot of issues on v14
